participants/op-mirrors.md

```diff
@@ -0,0 +1,66 @@
+# <op-mirrors>
+
+Before submitting a PR containing your team's project information:
+
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+Our project name is 'op-mirrors'. Our team is filecoin miner and also we're developing IPFS applications.
+
+## (1) Brief Project Description
+
+This is an app for all developers. App expect to save all software versions on the filecoin-network, so that developers can always find the version of the software they want. Of course, all software saved by the app is free and open source.
+
+## (2) Link to Project UI
+
+This is our project UI link :
+https://github.com/ptyz030529/mirror-web/blob/master/screenshot/racing-1.png
+https://github.com/ptyz030529/mirror-web/blob/master/screenshot/racing-2.png
+https://github.com/ptyz030529/mirror-web/blob/master/screenshot/racing-3.png
+THis is our project code link https://github.com/ptyz030529/mirror-web
+
+## (3) What does your application/UI do?
+
+The user can search the software , and download it. The user can also share the version of software to the filecoin-network by theirself.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+None
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+The data is download from the software download page, or upload by the other users.
```

Comment:
Can you specify a bit on what software will be loaded for this project?  Is the software public open-source applications or are there any license restrictions?  Will it be stored in unencrypted form?

Reply:
The possible include software: Linux OS, libreoffice, GIMP. We will limit the upload of software containing open source software license, such as GPL, MIT, Apache …… Encryption storage is not considered now.